simple/README.md

```diff
@@ -33,12 +33,6 @@
 2. Move into the directory, where you have downloaded the aforementioned resources in step 1.
 
 3. Deploy WSO2 Identity Server in your Kubernetes cluster.
-
-    * Deploy WSO2 Identity Server using Docker images from DockerHub.
```

Comment:
Remove these lines too 
  ```WSO2 product Docker images available at [DockerHub](https://hub.docker.com/u/wso2/) package General Availability (GA)
  versions of WSO2 products with no [WSO2 Updates](https://wso2.com/updates). ``` https://github.com/wso2/kubernetes-is/pull/323/files#diff-485807cbe3125f89817c993b149ea90950aa1a89c59c7eaf72c03eb6bf402de4R21